fleet_management_api/app.py

```
@@ -1,92 +1,159 @@
+    tenant_list = ",".join(f'"/customers/{name}"' for name in tenants)
+    payload = {
+        "Payload": '{{"group": [{tenant_list}]}}'.format(tenant_list=tenant_list),
+        "aud": "account",
+        "allowed-origins": ["test_client"],
+    }
+    private = _get_test_private_key()
+    if not private:
+        raise _MissingRSAKey("RSA private key is not set.")
+    try:
+        encoded = jwt.encode(payload, private, algorithm="RS256")
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)